js/msp/MSPCodes.js

```diff
@@ -76,9 +76,8 @@ var MSPCodes = {
     MSP_SET_PID_ADVANCED:           95,
     MSP_SENSOR_CONFIG:              96,
     MSP_SET_SENSOR_CONFIG:          97,
-    //MSP_SPECIAL_PARAMETERS:         98, // DEPRECATED
-    //MSP_SET_SPECIAL_PARAMETERS:     99, // DEPRECATED
-    //MSP_IDENT:                      100, // DEPRECTED
+    MSP_CAMERA_CONTROL:             98,
```

Comment:
I think `MSP_CAMERA_CONTROL` is introduced int BF firmware with the Camera OSD implementation. It is defined in PR https://github.com/betaflight/betaflight/pull/2727